pkgs/development/tools/rust/cargo-make/default.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  pname = "cargo-make";
+  version = "0.19.1";
+
+  src = fetchFromGitHub {
+    owner = "sagiegurari";
+    repo = pname;
+    rev = version;
+    sha256 = "1bv5gv5zd9h0yw7wb28r0c4z9wqa6n7ac4ipd855apwz1v7sm9n3";
+  };
+
+  cargoSha256 = "1l7fg1fq76g4bsyfffx338fcxvxb50qdwys8dww774k80dmdw4nh";
+  cargoPatches = [ ./0001-cargo-lock.patch ];
+
+  # Some tests fail because they need network access.
+  # However, Travis ensures a proper build.
+  # See also:
+  #   https://travis-ci.org/sagiegurari/cargo-make/jobs/530738212
```

Comment:
these links are going to get outdated on each version bump, can you delete them?